services/object/src/backends/index.js

```diff
@@ -39,8 +40,16 @@ class Backends {
         config,
       });
       this._backends.set(backendId, backend);
-      await backend.setup();
+      
+      try{
+        await backend.setup();
+      }catch(err){
+        this.monitor.reportError(new Error(err.message));
```

Comment:
```suggestion
        this.monitor.reportError(err.message);
```

This is the right idea, but with the changes in `aws.js` would not end up reporting any errors (because they'd be ignored in that file).